### PR TITLE
config: add flag to disable the E2EE support

### PR DESCRIPTION
**Pull request by saghul**
This is useful for testing insertable streams related issues.

Depends on https://github.com/jitsi/lib-jitsi-meet/pull/1238

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ var config = {
     //
 
     testing: {
+        // Disables the End to End Encryption feature. Useful for debugging
+        // issues related to insertable streams.
+        // disableE2EE: false,
```
Comment (paweldomas) [inline config.js:49]: is it possible to specify it through the URL params? is that something we would want? if yes the it needs to be added to the whitelist, right?

Comment (saghul) [reply config.js:49]: It's already possible to do so, because `config.testing` is whitelisted.

**Comment by paweldomas** [comment]
@saghul can you pull in the corresponding LJM version as part of this PR?

**Comment by saghul** [comment]
Done.

**Pull request merged**